src/views/model-viewer/ModelViewer.vue

```diff
@@ -50,13 +50,12 @@ export default {
     const topicGuid = route.query.topicGuid;
 
     // Initial plugins config
-    const pluginsConfig = PLUGINS_CONFIG;
-    Object.assign(pluginsConfig, {
+    const pluginsConfig = cloneDeep(PLUGINS_CONFIG);
```

Comment:
Either `cloneDeep` or make PLUGINS_CONFIG a function returning a new object every call. What do you think is best?